source/gui/settingsDialogs.py

```diff
@@ -1277,6 +1277,10 @@ def getType(self):
 		return DictionaryEntryDialog.TYPE_LABELS_ORDERING[typeRadioValue]
 
 	def onOk(self,evt):
+		if not self.patternTextCtrl.GetValue():
+			# Translators: This is an error message to let the user know that the pattern field in the dictionary entry is not valid.
+			gui.messageBox(_("A pattern is required."), _("Dictionary Entry Error"), wx.OK|wx.ICON_WARNING, self)

```

Comment:
Would it be a better user experience to disable the OK button when there is no text in the pattern field?